src/sa_web/templates/base.html

```diff
@@ -203,6 +206,9 @@ <h1 class="sidebar-header">{{ config.sidebar.panels.ticker.title }}<span class="
   <script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
   <script>window.jQuery || document.write('<script src="{{ STATIC_URL }}libs/jquery-1.10.2.js"><\/script>')</script><!-- FIXME: Maybe this should be pulled into the repo as a git submodule-->
 
+  <!-- jQuery date-time picker plugin -->
+  <script src="{{ STATIC_URL }}libs/jquery.datetimepicker.full.min.js"></script>
```

Comment:
Since this is a static asset, we should put this line along with our other library imports. For example, Line 244 would be a good spot for it.